### PR TITLE
adds trimal gt param to matrix_constructor.py

### PR DESCRIPTION
**Pull request by robert-ervin-jones**

### CODE REVIEW

File: phylofisher/matrix_constructor.py [hunk 1]
```diff
@@ -77,6 +78,7 @@ def run_snakemake(length_filter=False):
         f'--rerun-incomplete',
         f'--keep-going',
         f'--use-conda',
+        '--nolock'
```
Comment (atice) [inline phylofisher/matrix_constructor.py:81]: Just a question, does this need and f in front? a comma after?

**Review by atice — COMMENTED** [review]
Just the one question and then I can approve.

File: phylofisher/matrix_constructor.py (hunk 1 above)
Comment (robert-ervin-jones) [reply phylofisher/matrix_constructor.py:81]: No it doesn't

**Review by atice — APPROVED** [review]
Great!

**Pull request merged**